doc/README.md

```diff
@@ -137,20 +137,20 @@ All methods return on success a Promise with the server response parsed to an ob
 
 ```js
 endpoint
-  .client                                           // the httpClient instance
-  .endpoint                                         // name of the endpoint
-  .default                                          // The fetch client defaults
-  .find(resource, idOrCriteria, options)            // GET
-  .findOne(resource, id, criteria, options)         // GET
-  .post(resource, body, options)                    // POST
-  .update(resource, idOrCriteria, body, options)    // PUT
-  .updateOne(resource, id, criteria, body, options) // PUT
-  .patch(resource, idOrCriteria, body, options)     // PATCH
-  .patchOne(resource, id, criteria, body, options)  // PATCH
-  .destroy(resource, idOrCriteria, options)         // DELETE
-  .destroyOne(resource, id, criteria, options)      // DELETE
-  .create(resource, body, options)                  // POST
-  .request(method, path, body, options)             // method
+  .client                                                           // the httpClient instance
+  .endpoint                                                         // name of the endpoint
+  .default                                                          // The fetch client defaults
+  .find(resource, criteria, options, responseOutput)                // GET
```

Comment:
idOrCriteria was the better description as id, string and objects are allowed. better to update the docs to that instead of changing it to a somewhat missleading plain 'criteria' here